{"uuid": "4bf31644-7481-4260-b828-de625af2d578", "children": ["1307a33a-506f-4d74-819b-1a50b994033d"], "befores": [{"name": "setup", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.SessionNotCreatedException: Message: session not created\nfrom unknown error: no chrome binary at /usr/bin/google-chrome\nStacktrace:\n#0 0x5b8ead18610a <unknown>\n#1 0x5b8eace6bd20 <unknown>\n#2 0x5b8eacea1122 <unknown>\n#3 0x5b8eace9f569 <unknown>\n#4 0x5b8eaceeb5cb <unknown>\n#5 0x5b8eacedf213 <unknown>\n#6 0x5b8eacead2c5 <unknown>\n#7 0x5b8eaceae2ce <unknown>\n#8 0x5b8ead14fa4f <unknown>\n#9 0x5b8ead153b36 <unknown>\n#10 0x5b8ead13d4b7 <unknown>\n#11 0x5b8ead1542d1 <unknown>\n#12 0x5b8ead12479e <unknown>\n#13 0x5b8ead175098 <unknown>\n#14 0x5b8ead175298 <unknown>\n#15 0x5b8ead184d31 <unknown>\n#16 0x7719e4f7cac3 <unknown>\n\n", "trace": "  File \"/home/azizul/automation-4beats/venv/lib/python3.10/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n  File \"/home/azizul/automation-4beats/venv/lib/python3.10/site-packages/_pytest/fixtures.py\", line 1123, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"/home/azizul/automation-4beats/venv/lib/python3.10/site-packages/_pytest/fixtures.py\", line 895, in call_fixture_func\n    fixture_result = next(generator)\n  File \"/home/azizul/automation-4beats/tests/test_profile.py\", line 11, in setup\n    driver = webdriver.Chrome(options=options)\n  File \"/home/azizul/automation-4beats/venv/lib/python3.10/site-packages/selenium/webdriver/chrome/webdriver.py\", line 45, in __init__\n    super().__init__(\n  File \"/home/azizul/automation-4beats/venv/lib/python3.10/site-packages/selenium/webdriver/chromium/webdriver.py\", line 66, in __init__\n    super().__init__(command_executor=executor, options=options)\n  File \"/home/azizul/automation-4beats/venv/lib/python3.10/site-packages/selenium/webdriver/remote/webdriver.py\", line 212, in __init__\n    self.start_session(capabilities)\n  File \"/home/azizul/automation-4beats/venv/lib/python3.10/site-packages/selenium/webdriver/remote/webdriver.py\", line 299, in start_session\n    response = self.execute(Command.NEW_SESSION, caps)[\"value\"]\n  File \"/home/azizul/automation-4beats/venv/lib/python3.10/site-packages/selenium/webdriver/remote/webdriver.py\", line 354, in execute\n    self.error_handler.check_response(response)\n  File \"/home/azizul/automation-4beats/venv/lib/python3.10/site-packages/selenium/webdriver/remote/errorhandler.py\", line 229, in check_response\n    raise exception_class(message, screen, stacktrace)\n"}, "start": 1725977862647, "stop": 1725977863111}], "start": 1725977862647, "stop": 1725977863157}